.github/workflows/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,42 @@
+name: Bug Report
+description: Report a bug
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help everyone identify and fix the bug
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe your issue
+      placeholder: When I use ..., this happens
+    validations:
+      required: true
+  - type: textarea
+    id: code_bug
+    attributes:
+      label: Code to reproduce
+      placeholder: |
+        from qadence import *
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What is the expected result?
+      placeholder: I expected this to happen
+  - type: textarea
+    id: screenshots
```

Comment:
IDK how much you've thought about it, but I think it's more common (and in my experience more useful) to go with the layout
```
Short description
expected result
actual results
steps to reproduce
tracebacks
environment details (qadence version, operating  system, Python version, PyTorch version)
```

Reply:
@awennersteen Not much thought about it since I have not been exposed too much to them. But I just modified the layout according to your feedback, thanks!